snapcraft/plugins/copy.py

```diff
@@ -65,6 +67,13 @@ def schema(cls):
 
         return schema
 
+    def __init__(self, name, options, project):
+        super().__init__(name, options, project)
+
+        logger.warning("DEPRECATED: The 'copy' plugin's functionality "
+                       "has been replaced by the 'dump' plugin, and it will "
```

Comment:
> It is the same text used for tar-content

Ah, I was responding to @elopio, I think the text is fine. I wish github would the comment thread when emailing...